.github/workflows/percy.yml

```
@@ -1,99 +1,21 @@
-
-on:
-  push:
-    branches:
-      - master
-      - main
-  pull_request_target:
-    branches:
-      - master
-      - main
-
 jobs:
```

actionlint (1.7.11)

[error] "on" section is missing in workflow

(syntax-check)